src/arithmetic/string_funcs/string_funcs.c

```diff
@@ -35,9 +36,18 @@ SIValue AR_LEFT(SIValue *argv, int argc, void *private_data) {
 		// No need to truncate this string based on the requested length
 		return SI_DuplicateStringVal(argv[0].stringval);
```

Comment:
`SI_DuplicateStringVal` handles none ascii chars correctly ?

Reply:
what is the diff?